tensorflow/core/kernels/mkl_quantized_concat_op_test.cc

```diff
@@ -0,0 +1,230 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
```

Comment:
Could you please also wrap the file with the guards?
```c++
#if defined(INTEL_MKL) && defined(ENABLE_MKL)
...
#endif  // INTEL_MKL && ENABLE_MKL
```
Thank you!

Reply:
Done. Thanks!